setup.py

```diff
@@ -53,14 +53,14 @@ def include_dirs(self, dirs):
     "sounddevice==0.4.2",
     "rtmixer==0.1.3",
     "docutils==0.17.1",
-    "numpy==1.21.1",
+    "numpy>=1.21.1",
```

Comment:
Could we pin to the precise version that's needed ?
I guess that 1.21.5 or 1.22.1 ?

In the future I might move to a different build tool that supports locking the whole dependency tree, but in the meantime I prefer to have pinned versions to avoid unexpected build failures when a package gets silently upgraded to a newer version.